x/smart/stream_dialer_integration_test.go

```diff
@@ -107,7 +107,7 @@ fallback:
 		"request for A query failed: dial DNS resolver failed:",
 		`request for A query failed: receive DNS message failed: failed to get HTTP response: Post "https://mitm-software.badssl.com:443/dns-query": tls:`,
 		"🏃 running test: 'ss://Y2hhY2hhMjAtaWV0Zi1wb2x5MTMwNTprSzdEdHQ0MkJLOE9hRjBKYjdpWGFK@1.2.3.4:9999/?outline=1'",
-		"❌ Failed to start dialer: Psiphon: {PropagationChannelId: ID1, SponsorId: ID2, [...]} newPsiphonDialer failed: failed to start psiphon dialer:",
+		"❌ Failed to create dialer: [{psiphon: {DisableLocalHTTPProxy: true, DisableLocalSocksProxy: true, Establish…]: unsupported fallback type: psiphon",
```

Comment:
I would also test registering the psiphon config as part of this integration test, that way we'll be exercising
- registering the external module
- still trying to start a psiphon dialer (with an invalid config)

If the config is registered correctly you should start getting the previous error message again

Reply:
I meant literally updating the code of the integration test here to include registering the psiphon module. It can still test against the invalid config, but that way it tests that the registration/attempted connection itself is working as part of the ongoing CI.

Reply:
Done. This required some refactoring, since it created a dependency loop (`smart` -> `psiphon` -> `smart`). So now I have `smart/integrationstest`